solver/src/liquidity.rs

```diff
@@ -21,6 +21,7 @@ pub enum Liquidity {
 /// Basic limit sell and buy orders
 #[derive(Clone)]
 pub struct LimitOrder {
+    pub id: String,
```

Comment:
Would like a comment to explain what this is. Like "string used to identify order for debugging purposes". Or we could make this OrderUid if that is what it always is.

Reply:
I prefer using an opaque string here. In the future we might have other liquidity that behaves like a limit order but isn't attached to one of our off-chain orders. Will add the comment